tilerator/Dockerfile

```
@@ -14,31 +14,47 @@ RUN apt-get update && \
         nmap \
         netcat \
         redis-tools \
+        python3.5 \
+        python3-pip \
+        locales \
     && npm i npm@latest -g
 
 RUN git clone https://github.com/QwantResearch/tilerator.git /opt/tilerator \
     && cd /opt/tilerator \
     && git checkout 64b69c064aae15d474bd61da2c6bc35140f118f8 \
     && npm install --production
 
-RUN git clone https://github.com/QwantResearch/kartotherian_config.git /opt/kartotherian_config \
-    && cd /opt/kartotherian_config \
-    && git checkout c9fefaf505165c5a01bd24e3bc5295cfa9e73bde
+# install openmaptiles-tools
+RUN python3.5 -m pip install --upgrade pip \
+    && python3.5 -m pip install openmaptiles-tools
```

The version of openmaptiles-tools should be pinned
```suggestion
    && python3.5 -m pip install openmaptiles-tools==0.9.2
```